src/components/Divider/Divider.stories.tsx

```diff
@@ -35,7 +37,30 @@ export default {
                 ),
             },
         },
+        vertical: {
+            control: {
+                type: 'boolean',
+            },
+        },
     },
 } as Meta;
 
-export const Divider: StoryFn<DividerProps> = (args: DividerProps) => <DividerComponent {...args} />;
+export const Divider: StoryFn<DividerProps> = (args: DividerProps) => {
```

Comment:
There is a problem when you first load the story page: the `vertical` prop is false but it is rendering the divider in the vertical position... Idk if this a problem on the story or the component itself.
https://deploy-preview-1396--beta-fondue-components.netlify.app/?path=/story/components-divider--divider